### DISCUSSION TITLE
Extensión de navegador

### DISCUSSION THREAD

**Discussion by flbaynac** [IvanMass99/Calculadora-Gamer#1] [Ideas]
¿El proyecto podría exportarse a una extensión de navegador? Algo como steamcito pero para todas las plataformas, no se si se puede exportar svelte ya que no lo utilicé anteriormente, o hacer un frontend que solo se comunique con un backend en firebase para actualizar precios.

**Comment by IvanMass99** [comment]
¡Hola! La extensión está en la lista de ideas próximas a integrar como tambien una aplicación para móviles. Saludos!
